apps/web/README.md

```diff
@@ -34,3 +34,14 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+## Adding chains
+
+### Web
+
+Add the chain config to config/chains.ts
```

Comment:
```suggestion
Add the chain config to `config/chains.ts`.

```

Nit: Adding a line break, full stop, and code highlighting, because markdown currently renders as a single line.

<img width="879" alt="image" src="https://github.com/celo-org/faucet/assets/46296830/f43c2e00-aa7f-41f2-8e3e-2aeb23fc2bd6">